dashboards/MongoDB/MongoDB_ReplSet_Summary.json

```diff
@@ -2162,302 +2162,244 @@
         {
-          "activePatternIndex": 0,
-          "defaultPattern": {
-            "bgColors": "green|orange|red",
-            "bgColors_overrides": "0->green|2->red|1->yellow",
-            "clickable_cells_link": "",
-            "col_name": "Value",
-            "decimals": 2,
-            "defaultBGColor": "",
-            "defaultTextColor": "",
-            "delimiter": "|",
-            "displayTemplate": "_value_",
-            "enable_bgColor": false,
-            "enable_bgColor_overrides": false,
-            "enable_clickable_cells": false,
-            "enable_textColor": false,
-            "enable_textColor_overrides": false,
-            "enable_time_based_thresholds": false,
-            "enable_transform": false,
-            "enable_transform_overrides": false,
-            "filter": {
-              "value_above": "",
-              "value_below": ""
-            },
-            "format": "none",
-            "name": "Default Pattern",
-            "null_color": "darkred",
-            "null_textcolor": "black",
-            "null_value": "No data",
-            "pattern": "*",
-            "row_col_wrapper": "_",
-            "row_name": "Service Name",
-            "textColors": "red|orange|green",
-            "textColors_overrides": "0->red|2->green|1->yellow",
-            "thresholds": "70,90",
-            "time_based_thresholds": [],
-            "tooltipTemplate": "Series : _series_ <br/>Row Name : _row_name_ <br/>Col Name : _col_name_ <br/>Value : _value_",
-            "transform_values": "_value_|_value_|_value_",
-            "transform_values_overrides": "0->down|1->up",
-            "valueName": "current"
-          },
-          "default_title_for_rows": "Service Name",
-          "first_column_link": "/graph/d/mongodb-instance-summary/mongodb-instance-summary",
+          "collapsed": false,
           "gridPos": {
-            "h": 6,
+            "h": 10,
             "w": 24,
             "x": 0,
             "y": 61
           },
-          "hide_headers": false,
           "id": 1352,
-          "patterns": [
-            {
-              "bgColors": "green|orange|red",
-              "bgColors_overrides": "0->green|2->red|1->yellow",
-              "clickable_cells_link": "",
-              "col_name": "QPS",
-              "decimals": 2,
-              "defaultBGColor": "",
-              "defaultTextColor": "",
-              "delimiter": "|",
-              "displayTemplate": "_value_",
-              "enable_bgColor": false,
-              "enable_bgColor_overrides": false,
-              "enable_clickable_cells": false,
-              "enable_textColor": false,
-              "enable_textColor_overrides": false,
-              "enable_time_based_thresholds": false,
-              "enable_transform": false,
-              "enable_transform_overrides": false,
-              "filter": {
-                "value_above": "",
-                "value_below": ""
+          "pluginVersion": "9.2.18",
```

Comment:
Does this match grafana versions? If so, we are currently at 9.2.20.

Reply:
This PR was open before we updated to 9.2.20, let me update this.